HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.11.2
```

Comment:
We are still working on the `0.11.1` release.

Reply:
added to *.1